example/spec/parallel/asyncOperations/asyncOperationRunnerSpec.js

```diff
@@ -84,7 +84,7 @@ describe('The AsyncOperation task runner', () => {
       expect(parsedOutput.message).toContain('Function not found');
     });
     // Disabled pending resolution of CUMULUS-966
```

Comment:
Seems like we could remove these comments too?

Reply:
Fixed in 6787740b6